### PR TITLE
Introduce graphql doc update action

### PR DESCRIPTION
**Pull request by johannespfeiffer**
This action just runs `./update_docs.sh $(npx kontist token)` with the correct secret set up in the environment. If there is a change it should commit and push them to master.

### CODE REVIEW

**Comment by johannespfeiffer** [comment]
@hihuz please have a look. AFAIKS there is no easy way to test this inside of the PR

File: .github/workflows/grahpql_docs.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+          node-version: 12
+      - name: build docs
+        env:
+          KONTIST_CLIENT_ID: ${{secrets.DOCS_CLIENT_ID}}
+          KONTIST_CLIENT_SECRET: ${{secrets.DOCS_CLIENT_SECRET}}
+          KONTIST_USERNAME: ${{secrets.DOCS_USER}}
+          KONTIST_PASSWORD: ${{secrets.DOCS_PASSWORD}}
+          KONTIST_CLIENT_SCOPES: accounts,users,transactions,transfers,subscriptions,statements
+        run: |
+          ./update_docs.sh $(npx kontist token)
```
Comment (hihuz) [inline .github/workflows/grahpql_docs.yml:23]: should it be `./update_docs.sh "Bearer ${npx kontist token}"` 🤔 or maybe we should update the shell script to append the "Bearer " string directly 🤔

**Review by hihuz — COMMENTED** [review]
LGTM except for that one comment, what do you think?

File: .github/workflows/grahpql_docs.yml (hunk 1 above)
Comment (johannespfeiffer) [reply .github/workflows/grahpql_docs.yml:23]: Good point, will update `update_docs.sh`

**Pull request merged**